lms/djangoapps/grades/models.py

```diff
@@ -257,8 +257,14 @@ def save_grade(cls, **kwargs):
                     usage_key=usage_key,
                     defaults=kwargs,
                 )
+            log.info("persistent subsection grade successfully saved: {0}".format(unicode(grade)))
```

Comment:
I'll update the `unicode` method. I'm serializing `unicode(grade)` because this is a class method without access to `self`.